src/server.js

```diff
@@ -314,19 +314,26 @@ const PATH_NAME = process.env.NODE_ENV === "production" ?
   '~/gazelle-production/scripts/restartServers.sh' : process.env.NODE_ENV === "beta" ?
   '~/gazelle-beta/scripts/restartServers.sh' : __dirname + '/scripts/restartServers.sh';
 
+var isrestarted = false;
+
 editorTools.get('/restartserver', (req, res) => {
+
+  isrestarted = true;
+
   if (!process.env.NODE_ENV) {
-    // in dev mode
+    //in dev mode
     res.status(200).send("restarted");
```

Comment:
Just making sure, we don't restart the server in development right? 
So we are just checking if the server is up in development?